packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## `1.21.0`
+
+- Added unit tests for the FTP USS functionality [#1582](https://github.com/zowe/vscode-extension-for-zowe/pull/1582) and [1596](https://github.com/zowe/vscode-extension-for-zowe/pull/1596). Thanks @tiantn and JillieBeanSim
+- Added the new logger API. You can now use the Zowe Explorer API logger functions to standardize messages and specify message severity [#1518](https://github.com/zowe/vscode-extension-for-zowe/pull/1518). Thanks @katelynienaber
```

Comment:
This changelog item is accurate if the target audience is developers of the ZE FTP extension.

However, if we are addressing end users of the ZE FTP extension, this item might be a bit confusing. The new logger API is not something that the end user will be touching directly. Rather, they will just see that the ZE FTP extension's messages are standardized and have severity specified. I would suggest rewording to something like:
> Standardized user interface and log messages with severity specifications using Zowe Explorer API’s logger functions.